libevmasm/PeepholeOptimiser.cpp

```diff
@@ -75,7 +75,11 @@ struct SimplePeepholeOptimizerMethod
 
 struct Identity: SimplePeepholeOptimizerMethod<Identity>
 {
-	static bool applySimple(AssemblyItem const& _item, std::back_insert_iterator<AssemblyItems> _out)
+	static bool applySimple(
+		AssemblyItem const& _item,
+		std::back_insert_iterator<AssemblyItems> _out,
```

Comment:
```suggestion
		back_insert_iterator<AssemblyItems> _out,
```
The same goes for the other occurrences of this in the file, but I am not going to mark them all to avoid spamming the review with the same suggestion.

Reply:
I actually didn't changed that, I only added the `EVMVersion` there.

Reply:
Yeah, I noticed that, it's just that we have been removing the `std::` qualifier and this was a good opportunity to do it.